templates/monitoring/kube_state_metrics_3scale_alerts.yaml

```diff
@@ -94,15 +94,24 @@ spec:
         for: 5m
         labels:
           severity: critical
-      - alert: ThreeScalePodCount
```

Comment:
👍  Makes sense to remove this alert as there's a lot more specific alerts for the various components of 3scale

Reply:
agree